static/docs/user-guide/dvcignore.md

```diff
@@ -13,17 +13,17 @@ files handling. `.dvcignore` by design works similar way as `.gitignore` does.
 
```

Comment:
we need to update the paragraph above, it's too tailored to the "traversing" case